Patches/Vanilla Factions Expanded - Settlers/PawnKindDefs/PawnKinds_Settlers.xml

```diff
@@ -0,0 +1,66 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<Patch>
+
+  <Operation Class="PatchOperationFindMod">
+    <mods>
+      <li>Combat Extended</li>
```

Comment:
This *is* the Combat Extended mod, so this condition will always match. Did you mean to search for Vanilla Factions Expanded - Settlers?

```suggestion
      <li>Vanilla Factions Expanded - Settlers</li>
```

Reply:
Completely missed this lmao, thanks